repos/system_upgrade/el7toel8/actors/setuprhsmonrhel8/libraries/library.py

```diff
@@ -0,0 +1,40 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import mounting, rhsm
+from leapp.libraries.stdlib import CalledProcessError, api, run
+from leapp.models import TargetRHSMInfo, UsedTargetRepositories
+
+
+def set_release():
+    """Set the RHSM release to the target RHEL 8 minor version."""
+    info = next(api.consume(TargetRHSMInfo), None)
+    if info and info.release:
+        rhsm.set_release(mounting.NotIsolatedActions(base_dir='/'), info.release)
+        api.current_logger().debug('Release {0} successfully set.'.format(info.release))
+    else:
+        api.current_logger().debug('Skipping setting the RHSM release due to the use of LEAPP_DEVEL_SKIP_RHSM.')
+
+
+def enable_repos():
+    """Enable all the RHEL 8 repositories that have been used for the upgrade transaction."""
+    try:
+        run(get_submgr_cmd())
+    except CalledProcessError as err:
+        api.current_logger().error('Could not enable RHEL 8 repositories:\n {0}'.format(str(err)))
```

Comment:
Would be better to print here just warning that some repos have not been enabled - with info that in case of custom repositories it is expected and user has to enable those manually.

Reply:
Done.